source/_docs/select-plan.md

```diff
@@ -1,34 +1,48 @@
-3. Once this is done, an email is sent to the account of the user who is associated with the credit card.
+<div class="alert alert-info" role="alert">
+<h3 class="info">Note</h3>
+<p>Only the Site Owner, or for sites owned by an Organization - the Organization's Administrators, can manage a site's plan. Users without the permission to manage a site's plan will not see the options discussed below. For additional details, see <a href="/docs/change-management">Role-Based Permissions & Change Management</a>.</p></div>
 
-<!--![Select a plan from dashboard](/source/docs/assets/images/select-a-plan-and-billing.png)-->  
 
-## Go Live
-Now that you have selected a plan, continue the process of going live by [setting up a domain on your site](/docs/going-live).
+## View Current Plan
+The site's current service level is shown within a green label on the Site Dashboard:
 
-## View Your Site's Plan
+![confirm new plan](/source/docs/assets/images/dashboard/confirm-plan-dashboard.png)
 
-Identifying your site's service level is quick and easy. You can find this in the green label next your site's name.
+## Select Payment Method and Plan
+1. Select a payment method within **Settings** > **Billing**, then click **Update Payment Method**. You can choose from existing cards, add a new card or invite the site owner to pay.
+2. Select the desired plan from **Settings** > **Plan**, then click **Update Plan**.
+3. As the site owner, you will receive an email confirming the change to the site.
+4. After the site billing is processed, you will receive an updated invoice.
 
-![confirm new plan](/source/docs/assets/images/dashboard/confirm-plan-dashboard.png)
+## Upgrade or Downgrade Plan
+Manage a site's plan from the Site Dashboard within **Settings** > **Plan**. Select the desired plan and click **Update Plan**. Plan changes take immediate effect.
 
-<div class="alert alert-info" role="alert">
-<h3 class="info">Note</h3>
-<p>If you downgrade from a paid plan to sandbox, the <a href="/docs/">domains</a> you've added in your Live environment will be removed. You will need to re-add domains if you decide to return to a paid plan.</p></div>
+### Downgrade to Sandbox
+The following should be considered before downgrading your site from a paid plan to Sandbox:
+
+* All domains added to Pantheon environments will be removed. If you decide to return to a paid plan in the future, you will need to add the domains again in desired environments.
+* HTTPS will be disabled and certificates will be removed. If you decide to re-enable HTTPS in the future, you must re-upload your certificate, intermediary certificates, and your key.
+* A prorated credit will be issued to your billing account.
+
+### Downgrade or Upgrade to Personal
+The following should be considered before downgrading or upgrading to a Personal plan:
+
+* Redis and Solr will be disabled.
```

Comment:
@lauren-kelly I'd like to include steps to disable Redis and Solr within WordPress and Drupal. Can you help identify those steps?